script/docker_test.rb

```diff
@@ -1,12 +1,44 @@
 # frozen_string_literal: true
 
-# This script is run in the discourse_test docker image
-# Available environment variables:
-# => NO_UPDATE        disables updating the source code within the discourse_test docker image
-# => COMMIT_HASH      used by the discourse_test docker image to load a specific commit of discourse
-#                     this can also be set to a branch, e.g. "origin/tests-passed"
-# => RUN_SMOKE_TESTS  executes the smoke tests instead of the regular tests from docker.rake
-# See lib/tasks/docker.rake and lib/tasks/smoke_test.rake for more information
```

Comment:
No need for this now that we switch to `OptionParser` and have the `--help` option.